orderedmap_test.go

```diff
@@ -102,17 +102,40 @@ func TestBasicFeatures(t *testing.T) {
 
 	// check iterations again
 	i = 0
-	for pair := om.Oldest(); pair != nil; pair = pair.Next() {
+	for pair := om.First(); pair != nil; pair = pair.Next() {
```

Comment:
Nope, sorry. We still need to check that the "historic" version works. Let's just test both here